variablegroups/AAT-AKS-Common.json

```diff
@@ -46,7 +46,7 @@
       "value": "[\"10.96.139.254\"]"
     },
     "Subscription.keyvaultName": {
-      "value": "cftapps-stg"
+      "value": "cftapps-$(Env.Override.Name)"
     },
     "Subscription.ServiceConnection.Name": {
       "value": "DCD-CFTAPPS-STG"
```

Comment:
are we using STG's service connection name or should we have a different one for prod?